src/bouncerbot/response.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@thewanderingeditor Can you take a look at this content? I believe we had a small discussion on whether we wanted to do some changes on the content, but unsure on whether we moved forward there.